background/get-userscripts.js

```diff
@@ -400,14 +400,21 @@ function matchesIf(injectable, settings) {
 function userscriptMatches(data, scriptOrStyle, addonId) {
   if (scriptOrStyle.if && !matchesIf(scriptOrStyle, scratchAddons.globalState.addonSettings[addonId])) return false;
 
-  const url = data.url;
-  const parsedURL = new URL(url);
+  let _url = data.url;
+  let _parsedURL = new URL(_url);
+  if (_parsedURL.origin === "https://scratchfoundation.github.io" || _parsedURL.port === "8601") {
+    // Run addons on scratch-gui
+    _url = "https://scratch.mit.edu/projects/104/editor";
```

Comment:
It makes more sense to run as a logged-out editor page.

Reply:
Good point! I will change this to `https://scratch.mit.edu/projects/editor/`